tools/accuracy_checker/configs/README.md

```diff
@@ -0,0 +1,96 @@
+
+Example:
+
+```yaml
+evaluations:
+  - name: model_name
+    module: name_of_class_with_custom_evaluators
+    module_config:
+      network_info:
+        encoder: {}
+
+        decoder:
+          adapter: adapter_name
+
+      launchers:
+        - framework: dlsdk
+
+      datasets:
+        - name: dataset_name
+```
+
+## Location
+
+Predefined configuration file `accuracy-check.yml` for each Open Model Zoo model can be found in the model directory.
+
+`<model_name>.yml` file, which is located in current `configs` folder, is a link to `accuracy-check.yml` for `<model_name>` model.
+
+Example:
+
+[alexnet.yml](alexnet.yml) is a link for configuration file [accuracy-check.yml](../../../models/public/alexnet/accuracy-check.yml) for [alexnet](../../../models/public/alexnet/alexnet.md) model.
+
+## Options
+
+1. To run configuration specify the path to the required configuration file to `-c, --config` command line.
+2. Configuration files don't contain paths to used models and weights. The model and weights are searched automatically by name of model in path specified in `-m, --models` command line option.
+3. There is global configuration file with dataset conversion parameters which is used to avoiding duplication. Global definitions will be merged with evaluation config in the runtime by dataset name. You can use global_definitions to specify path to this file via command line arguments `-d, --definitions`.
+4. The path relative to which the `data_source` is specified can be provided via  `-s, --source` command line. If you want to evaluate models using well-known datasets, you need to organize folders with validation datasets in a certain way. More detailed information about dataset preparation you can find in [Dataset Preparation Guide](../../../datasets.md).
+5. The path relative to which the  `annotation` and `dataset_meta` are specified can be provided via `-a, --annotations` command line.   Annotation and dataset_meta (if required) will be stored to this directory after annotation conversion step if they do not exist and can be used for the next running to skip annotation conversion. Detailed information about annotation conversion you can find in [Annotation Conversion Guide](../accuracy_checker/annotation_converters/README.md).
+6. Some models can have additional files for evaluation (for example, vocabulary files for NLP models), generally, named as model attributes. The relative paths to model specific attributes(vocabulary files, merges files, etc.) can be provided in the configuration file, if it is required. The path prefix for them should be passed through `--model_attributes` command line option (usually, it is the model directory).
+7. To specify devices for infer use `-td, --target_devices` command line option. Several devices should be separated by spaces (e.g. -td CPU GPU).
+
```

Comment:
I think we need also to add info about target framework, because public models config may contains also running for source framework and in some cases user can have not installed or does not want run it.